projects/cras/Dockerfile

```diff
@@ -55,3 +59,6 @@ RUN mkdir -p /tmp/alsa-build && cd /tmp/alsa-build && \
 RUN cd $SRC && git clone https://chromium.googlesource.com/chromiumos/third_party/adhd
 WORKDIR adhd/cras
 COPY build.sh $SRC/
+
+# Copy dsp.ini.sample to /etc/cras for the fuzzer
+RUN mkdir -p /etc/cras && cp /src/adhd/cras-config/dsp.ini.sample /etc/cras
```

Comment:
is this a build time dependency? If it's needed during runtime, it will not be available on the fuzzing VMs

Reply:
It'll be used in runtime but let's remove it for now. This won't affect existing fuzzer function.